### ISSUE TITLE
A11y: [Chats] Search input field lacks a programmatically associ...

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-android#6389]
- **Checkpoint ID:** 11.1.3.1d/4.1
- **Component:** Chats

### Issue Description
Search input field lacks a programmatically associated label.

### Recommendation
Associate visible label with text field via accessibilityLabel.

--- 
*Report Date: 25.02.2026*
